src/generator/AutoRest.Ruby.Azure/Model/CompositeTypeRba.cs

```diff
@@ -12,6 +16,10 @@ namespace AutoRest.Ruby.Azure.Model
     /// </summary>
     public class CompositeTypeRba : CompositeTypeRb
     {
+        public static readonly Regex nameRegEx         = new Regex(@"^(RESOURCE|SUBRESOURCE)$", RegexOptions.IgnoreCase);
```

Comment:
should this be called resourceOrSubresourceRegEx or something like that? I'm not sure nameRegEx describes the purpose of this regex